nixos/roles/mailserver.nix

```diff
@@ -45,12 +45,12 @@ let
       example = "mail.example.com";
     };
 
-    policydSPFExtraSkipAddresses = mkOption {
+    spfSkipAddresses = mkOption {
       type = with types; listOf str;
       description = ''
-        Extra addresses policyd should skip in SPF checks.
+        Extra addresses rspamd should skip in SPF checks.
         Local addresses are always skipped.
-        This extends the `skip_addresses` setting in policyd-spf.conf.
+        Format: IP address with optional CIDR notation (rspamd radix map)
       '';
       default =
```

Comment:
We now loose 

```
"127.0.0.0/8"
"::ffff:127.0.0.0/104"
"::/64"
```

have you considered this and is this intentional?

Reply:
Yes, "127.0.0.0/8" and "::/64" are already ignored altogether by rspamd [via `local_addrs`](https://github.com/flyingcircusio/fc-nixos/blob/0c38bcb45774d89849210b402f9293ea376f9422/nixos/services/mail/rspamd.nix#L47).
I had additionally tried to add "::ffff:127.0.0.0/104", but then rspamd was showing a warning about duplicate addresses in the radix map until I removed it again.